game-core/src/main/java/games/strategy/engine/framework/startup/launcher/ServerLauncher.java

```diff
@@ -275,8 +275,8 @@ private void warmUpCryptoRandomSource() {
     new Thread(() -> {
       try {
         serverGame.getRandomSource().getRandom(gameData.getDiceSides(), 2, "Warming up crypto random source");
-      } catch (final RuntimeException re) {
-        re.printStackTrace(System.out);
+      } catch (final RuntimeException e) {
+        ClientLogger.logQuietly("Failed to warm up crypto random source", e);
```

Comment:
I'm not sure, is this shared code between headless and non-headless? (or non-headless only?)

Reply:
I believe this class is used by both the headed and headless game servers.  Other methods in this class are already using `ClientLogger` (not claiming that's correct, but it's precedent).